trash-bin cli has been exteneded by the list and restore commands (#7917)
owncloud/ocis: services/storage-users/README.md

*   `STORAGE_USERS_PURGE_TRASH_BIN_PROJECT_DELETE_BEFORE` has a default value of `30 days`, which means the command will delete all files older than `30 days`. The value is human-readable, valid values are `24h`, `60m`, `60s` etc. `0` is equivalent to disable and prevents the deletion of `project space` trash-bin files.

#### List and Restore Trash-Bins Items
```bash
NAME:

>    ocis storage-users trash-bin list - Print a list of all trash-bin items in a space.

Reviewer: ```suggestion
   ocis storage-users trash-bin list - Print a list of all trash-bin items for a space.
```

USAGE:
   ocis storage-users trash-bin list command [command options] ['spaceID' required]
```

```bash
NAME:
   ocis storage-users trash-bin restore-all - Restore all trash-bin items for a space.

USAGE:
   ocis storage-users trash-bin restore-all command [command options] ['spaceID' required]

COMMANDS:
   help, h  Shows a list of commands or help for one command

OPTIONS:

>    --option value, -o value  The restore option defines the behavior if the file with the same name that needs to be restored already exists in a space. Expected values [skip] [replace] [keep-both] (default: The default value is [skip] restoring exists file. )

Reviewer: ```suggestion
   --option value, -o value  The restore option defines the behavior for a file to be restored, where the file name already already exists in the target space. Expected values are: 'skip', 'replace' 'keep-both' (default: The default value is 'skip' restoring exists file.)
```

```

```bash
NAME:
   ocis storage-users trash-bin restore - Restore a trash-bin item by ID.

USAGE:

>    ocis storage-users trash-bin restore command [command options] ['spaceID' required] ['itemID' required]

Reviewer: We need a description how to get the required `spaceID` and `itemID`.

COMMANDS:
   help, h  Shows a list of commands or help for one command

OPTIONS:

>    --option value, -o value  The restore option defines the behavior if the file with the same name that needs to be restored already exists in a space. Expected values [skip] [replace] [keep-both] (default: The default value is [skip] restoring exists file. )

Reviewer: ```suggestion
   --option value, -o value  The restore option defines the behavior for a file to be restored, where the file name already already exists in the target space. Expected values are: 'skip', 'replace' 'keep-both' (default: The default value is 'skip' restoring exists file. )
```
